src/screens/Onboarding/StepFinished.tsx

```diff
@@ -160,7 +160,16 @@ export function StepFinished() {
     onboardDispatch({type: 'finish'})
     track('OnboardingV2:StepFinished:End')
     track('OnboardingV2:Complete')
-    logEvent('onboarding:finished:nextPressed', {})
+
+    let avatarResult = 'default'
+    if (profileStepResults.isCreatedAvatar) {
+      avatarResult = 'created'
+    } else if (profileStepResults.image) {
+      avatarResult = 'uploaded'
+    }
+    logEvent('onboarding:finished:nextPressed', {
+      avatarResult,
+    })
```

Comment:
we should only call `logEvent` if the gate is true I think, otherwise everyone including people who are in old onboarding will be logging this.